tests/test_helpers.py

```diff
@@ -7,6 +7,33 @@
 
 dash = "-"
 helper_map = [
+    (["helper", "attestations", "dmref15", "-b", "2019-09-01", "-e", "2019-11-01", "--no-plot"],
```

Comment:
we will almost certainly have to pass in a date for testing, or the code will default to `today` on the day the test is run and the behavior will change over time.  Not sure, but that is usually the case.

Reply:
If no begin/end_date is passed, the program just gets the begin/end_date from get_dates(grant)['begin/end_date']